wordpress.yaml

```diff
@@ -1,7 +1,149 @@
 ---
 - name: Deploy Wordpress
   hosts: all
+  vars_files:
+    - /vagrant/variables.yaml
+    
   tasks:
-
-    - name: message to remove
-      command: echo 'Here you gonna put all tasks and roles that gonna be runing on the VM'
+    - name: Create a login user
+      become: yes
+      user:
+       name: "{{ user_name }}" 
+       password: "{{ user_pwd | password_hash('sha512') }}"
+       groups: 
+        - sudo
+       state: present
+       shell: "{{ user_shell }}"        
+       system: no             
+       createhome: yes        
+       home: "{{ user_home }}" 
+      
+    - name: Update the System
+      apt:
+        update_cache: yes
+    
+    - name: Install LEMP Packages
+      apt:
+        name:
+            - nginx
+            - mysql-server
+            - python3-mysqldb
+            - php7.4-cli
+            - php7.4-fpm
+            - php7.4-mysql
+            - php7.4-json
+            - php7.4-opcache
+            - php7.4-mbstring
+            - php7.4-xml
+            - php7.4-gd
+            - php7.4-curl
+            
+    - name: Allow NGINX on Firewall   
+      ufw:
+        rule: allow
+        name: "Nginx Full"
+        
+    - name: Create SSL-Certificate
+      shell: |
+                if [ ! -f /etc/ssl/certs/nginx-selfsigned.crt ]; then
```

Comment:
ansible has great feature to run this if file not exists. Could you check it out?

Reply:
done